README.md

```diff
@@ -84,23 +78,22 @@ Tibbles have a refined print method that shows only the first 10 rows, and all t
 ``` r
 library(nycflights13)
 flights
-#> Source: local data frame [336,776 x 16]
```

Comment:
I'm not sure I like getting rid of this line - it is nice to have an up front summary, but it does cost two lines.